string-manipulation-2/src/ransom-case.js

```diff
@@ -1,16 +1,15 @@
 /* exported ransomCase */
 function ransomCase(string) {
   var ransomWord = '';
-  var currentLetter = '';
-  var stringHolder = '';
+  var stringHolder;
   for (var i = 0; i < string.length; i++) {
     if (i % 2 === 0) {
       stringHolder = string.toLowerCase();
+      ransomWord = ransomWord + stringHolder.slice(i, i + 1);
```

Comment:
How about this: `ransomWord += string[i].toLowerCase()`

That should remove the need for the `stringHolder` variable. Please update your else block too!